spec/std/array_spec.cr

```diff
@@ -1497,6 +1497,33 @@ describe "Array" do
       a.@offset_to_buffer.should eq(0)
       a.@buffer.should eq(buffer)
     end
+
+    it "repeated unshift/shift does not exhaust memory" do
+      a = [] of Int32
+      10000.times do
```

Comment:
Do such many iterations bring any reasonable benefit?
I figure even two passes would be sufficient proof. Or make it three to be safe.